src/sentry/api/serializers/models/integration_feature.py

```diff
@@ -9,6 +9,4 @@ def serialize(self, obj, attrs, user):
             "description": obj.description.strip(),
             # feature gating work done in getsentry expects the format 'featureGate'
             "featureGate": obj.feature_str(),
-            "targetId": obj.target_id,
```

Comment:
It looks like this is breaking tests.

Reply:
yep, just noticed that as well -- fixing now